nltk/parse/dependencygraph.py

```diff
@@ -321,10 +326,15 @@ def extract_10_cells(cells):
                         'CoNLL(10) or Malt-Tab(4) format'.format(cell_number)
                     )
 
-            word, lemma, ctag, tag, feats, head, rel = cell_extractor(cells)
+            line_index, word, lemma, ctag, tag, feats, head, rel = cell_extractor(cells)
```

Comment:
But we need to be careful here, as we might get a cell_extractor which doesn't take index as an argument.